storage/storage_zmq_client.c

```diff
@@ -0,0 +1,107 @@
+//  Weather update client
+//  Connects SUB socket to tcp://localhost:5556
+//  Collects weather updates and finds avg temp in zipcode
+
+#include "scylla_api.h"
+#include "zhelpers.h"
```

Comment:
Is `zhelpers.h` really useful for development? I don't see it is necessary.

Reply:
No, only one function been used in zhelpers.h. I will remove it.